### PR TITLE
Add Cape Verde

### PR DESCRIPTION
**Pull request by josepaiva94**
Added Cape Verde

### CODE REVIEW

File: packages/cape-verde/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# 🇨🇻 Cape Verde
+
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-blue.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+Contains all the provinces of Cape Verde:
```
Comment (VictorCazanave) [inline packages/cape-verde/README.md:5]: It looks like they are called "municipalities" (source: https://en.wikipedia.org/wiki/Administrative_divisions_of_Cape_Verde)

File: packages/cape-verde/cape-verde.svg [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<svg
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 860 793"
+		width="859.93915"
+		height="792.58252"
```
Comment (VictorCazanave) [inline packages/cape-verde/cape-verde.svg:5]: Could you remove the `width` and `height` attributes please?

File: packages/cape-verde/cape-verde.svg [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+			name="Brava"
+			id="br"/>
```
Comment (VictorCazanave) [inline packages/cape-verde/cape-verde.svg:11]: Are these `id`s common short names?

File: packages/cape-verde/package.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+	"name": "@svg-maps/cape-verde",
+	"version": "1.0.1",
```
Comment (VictorCazanave) [inline packages/cape-verde/package.json:3]: `"version": "1.0.0"`

File: packages/cape-verde/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	"description": "Map of Cape Verde",
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/VictorCazanave/svg-maps/tree/master/packages/cape-verde"
+	},
+	"keywords": [
+		"svg",
+		"map",
+		"cape-verde"
```
Comment (VictorCazanave) [inline packages/cape-verde/package.json:13]: `"cape verde"`

File: packages/cape-verde/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
+# 🇨🇻 Cape Verde
```
Comment (VictorCazanave) [inline packages/cape-verde/README.md:1]: Could you also add it in the main README please?

**Review by VictorCazanave — COMMENTED** [review]
Thanks a lot for your contribution @josepaiva94 ! 😃 

Just some small things to fix and 1 question, then I'll glad to merge it.

File: packages/cape-verde/cape-verde.svg (hunk 3 above)
Comment (josepaiva94) [reply packages/cape-verde/cape-verde.svg:11]: They are not common short names, they are acronyms. When used in the context of administrative divisions of Cape Verde, they clearly identify a division.

**Comment by josepaiva94** [comment]
All items solved. Thanks a lot for taking time to review!

**Comment by VictorCazanave** [comment]
Thank to you!

@all-contributors please add @josepaiva94 for code

**Pull request merged**